### PR TITLE
feat : Merge meeds-io/MIPs#105

### PR DESCRIPTION
**Pull request by rdenarie**
With the merge of this mip, actions needs an icon on trigger list This commit add the js extension to add the icon

### CODE REVIEW

File: webapp/src/main/webapp/engagementCenterExtensions/extensions.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * version 3 of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
```
Comment (azayati) [inline webapp/src/main/webapp/engagementCenterExtensions/extensions.js]: exo header

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/engagementCenterExtensions/extensions.js (hunk 1 above)
Comment (rdenarie) [reply webapp/src/main/webapp/engagementCenterExtensions/extensions.js]: done

File: webapp/src/main/webapp/engagementCenterExtensions/extensions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 eXo Platform SAS.
```
Comment (azayati) [inline webapp/src/main/webapp/engagementCenterExtensions/extensions.js]:
```suggestion
 * Copyright (C) 2024 eXo Platform SAS.
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**